### DISCUSSION TITLE
embed, encode, attend, predict [Source code link]

### DISCUSSION THREAD

**Discussion by VaibhaveS** [explosion/spaCy#11922] [Help: Coding & Implementations]
I was going through this article [https://explosion.ai/blog/deep-learning-formula-nlp#embed]

Where can I find the implementation for these steps? I am particularly interested in the encode step.

It would be great if someone can link the source code for the same, thanks.

**Comment by polm** [comment]
The embed and encode steps are covered by the [tok2vec architectures](https://spacy.io/api/architectures#Tok2Vec).
